### PR TITLE
magic-url-e2e-issue

### PR DESCRIPTION
**Pull request by jablonnc**
Addresses an issue with our E2E tests mentioned in [this thread](https://jptrone.slack.com/archives/C016KQ5DEKX/p1699461757233239).

### CODE REVIEW

File: .github/actions/frontend/runtime/e2e_run/action.yml [hunk 1]
```diff
@@ -148,8 +153,8 @@ runs:
         CYPRESS_ENVIRONMENT_NAME: jupiterone-dev
         GITHUB_TOKEN: ${{ inputs.github_token }}
         MIGRATION_NUMBER: ${{ inputs.migration_number }}
-        PR_NUMBER: ${{ inputs.commit_info_pr_number || github.event.pull_request.number }}
```
Comment (jablonnc) [inline .github/actions/frontend/runtime/e2e_run/action.yml:151]: This was the primary issue as `github.event.pull_request.number` doesn't exists in the context of the `issue_comment` event

Comment (jablonnc) [inline .github/actions/frontend/runtime/e2e_run/action.yml:151]: This was the primary issue as github.event.pull_request.number doesn't exists in the context of the issue_comment event. So the test run was referencing an invalid magic url.

File: .github/actions/frontend/runtime/e2e_run/action.yml [hunk 2]
```diff
@@ -20,21 +20,26 @@ inputs:
   commit_info_sha:    
     description: 'The commit sha associated with the repo that triggered the e2e_run'
     type: string
+    required: true
```
Comment (Sivli-Embir) [inline .github/actions/frontend/runtime/e2e_run/action.yml:23]: Will this break all the current E2E users?

Comment (jablonnc) [reply .github/actions/frontend/runtime/e2e_run/action.yml:23]: No, the only consumers of this are our workflows. The two in question now pass this in as an input.

**Review by Sivli-Embir — APPROVED** [review]

**Pull request merged**